.github/workflows/main.yml

```diff
@@ -156,9 +155,8 @@ jobs:
 
           echo "::set-output name=commitHash::$(git rev-parse --short=9 HEAD)"
 
-      - name: 'Install Xcode 14 Beta'
-        if: ${{ startsWith(github.ref, 'refs/tags/') && github.event_name != 'pull_request' && env.HAVE_CODESIGN_IDENTITY == 'true' && env.HAVE_XCODE_URL == 'true' }}
-        run: xcversion install "14 beta 4" --url=${{ secrets.XCODE_DOWNLOAD_URL }}
+      - name: 'Swwitch to Xcode 14 Beta'
```

Comment:
Swwitch

Reply:
Do we need the word "Beta" here? Eventually, Xcode 14 will not be beta, and I don't think the xcode-select invocation will change.

Reply:
One it is out of beta it will probably become the default and the entire command can be removed.

Reply:
They don't make it the default right away. May 17, they added Xcode 13.4, but 13.3.1 was default. June 9, they added Xcode 13.4.1, but 13.3.1 remained the default. June 28, they finally announce that Xcode 13.4.1 will become the default on July 11.

Not blocking, but wanted to clarify expected update timetables.

Reply:
We may still have to update this manually whenever Xcode 14 stable comes out due to point releases. So we can remove the "Beta" whenever that happens, and then fully remove it once 14 becomes the default.